tailwind.config.js

```diff
@@ -17,9 +17,11 @@ export default {
 			'off-white': '#fcfaf6',
 			dust: '#dbd9d6',
 			grey: '#c0bbc4',
+			'light-grey': '#ced4da',
```

Comment:
This change is expected and if yes, isn't the corresponding variables missing in `colors.scss`?

Reply:
Yes, it is one of the color of the pallet, but I didn't want to add it as variable color, since my plan is to slowly deprecate the CSS styles. In this way, the color is available only for Tailwind classes.

Of course, if at some point it is absolutely necessary to have it for CSS, I will add it

Reply:
I would add it to the class because we gonna start having some variables set and some not.
Not against deprecating the style class but either we do it or not if you know what I mean?

Reply:
Consistency, make sense!